### PR TITLE
add Googletranslation

### PR DESCRIPTION
**Pull request by x0rzkov**
based on an (unofficial) Googletrans: Free and Unlimited Google translate API for Python. Translates totally free of charge.

### CODE REVIEW

File: twint/tweet.py [hunk 1]
```diff
@@ -114,6 +114,10 @@ def Tweet(tw, config):
         try:
             ts = translator.translate(t.tweet)
             t.translate = ts.text
+            t.trans_src = ts.src
+            t.trans_dest = ts.dest
         except:
```
Comment (pielco11) [inline twint/tweet.py]: You should specify the correct exception to handle

**Comment by pielco11** [comment]
Also the support for Elasticsearch is missing, please check `elasticsearch.py` and do as I did for `tweet` field

File: twint/tweet.py (hunk 1 above)
Comment (x0rzkov) [reply twint/tweet.py]: what do you mean ?

src by default is **en** and dest=**auto**, so the translate method will trigger an exception

ref. https://py-googletrans.readthedocs.io/en/latest/#googletrans.Translator.translate

Comment (x0rzkov) [reply twint/tweet.py]: Can you provide an example ? (I started python 1 month ago ^^ with Twint, I am more Golang like)

Comment (pielco11) [reply twint/tweet.py]: Ok so let's say that you have a dict like `dict1 = {'first_key':'first_value', 'second_key':'second_value'}`. Now if you do something like `third_value = dict1['third_key']`, Python will raise a  `KeyError` exception since in `dict1` the key `third_key` is not defined. To handle this exception you have to do something like

```python
try:
    third_value = dict1['third_key']
except KeyError:
    print("dict1 has not third_key as key")
```

Now if you don't specify `except KeyError`, `except:` will handle every possible exception, like `KeyboardInterrupt` (when you hit `CTRL + C`)

Basically you should check that library and see which exceptions could be raised, and handle that specific one

File: twint/tweet.py [hunk 2]
```diff
@@ -114,10 +114,11 @@ def Tweet(tw, config):
...
     if config.Translate == True:
         try:
-            ts = translator.translate(t.tweet)
+            # config.Translate
+            ts = translator.translate(text=t.tweet, dest=config.TranslateDest)
             t.translate = str(ts.text)
             t.trans_src = str(ts.src)
             t.trans_dest = str(ts.dest)
-        except Exception as e:
+        except KeyError:
```
Comment (pielco11) [inline twint/tweet.py]: You sure that `KeyError`is the right error to handle? Usually it happens with `dict` variables, my previously described way to handle exceptions was just an example and might be the right exception to handle

**Comment by pielco11** [comment]
It breaks and returns 

```python
File "/home/travis/build/twintproject/twint/twint/storage/write_meta.py", line 36, in tweetData
    "trans_dest": t.trans.dest
AttributeError: 'tweet' object has no attribute 'trans'
```

**Comment by x0rzkov** [comment]
> It breaks and returns
> 
> ```python
> File "/home/travis/build/twintproject/twint/twint/storage/write_meta.py", line 36, in tweetData
>     "trans_dest": t.trans.dest
> AttributeError: 'tweet' object has no attribute 'trans'
> ```

My bad, it was a stupid typo...

**Comment by pielco11** [comment]
Am I missing something?

immagine

**Comment by x0rzkov** [comment]
> Am I missing something?
> 
> immagine

I am on it !

**Comment by x0rzkov** [comment]
```
import twint

c = twint.Config()
c.Username = "noneprivacy"
c.Limit = 100
c.Elasticsearch = "localhost:9200"
c.Store_csv = True
c.Output = "none.csv"
c.Lang = "en"
c.Translate = True
c.TranslateDest = "it"
twint.run.Search(c)
```

Tested successfully with sqlite3, csv and elasticsearch

**Comment by pielco11** [comment]
- [x] Elasticsearch
- [x] CSV
- [x] JSON
- [x] SQLite

**Comment by pielco11** [comment]
If you want to write a few lines for the wiki, just comment and I'll add them!
